debian/tesseract-training.install

```diff
@@ -0,0 +1,35 @@
+src/training/tesstrain.sh usr/share/tesseract-ocr/
+src/training/language-specific.sh usr/share/tesseract-ocr/
+src/training/tesstrain_utils.sh usr/share/tesseract-ocr/
```

Comment:
I think that `tesstrain.sh` and the other two shell scripts will be replaced by the already existing Python scripts in the future. So for Tesseract 5, adding those Python scripts might be better.

@amitdo, @Shreeshrii, what do you think?

Reply:
@stweil I thought you were planning to move the Python training scripts to tesstrain repo.

Reply:
That's correct, but I still don't know whether people would miss the current shell scripts for training which would be removed then.